demo/server.js

```diff
@@ -34,6 +34,8 @@ app.use(function (req, res) {
         response = db.get('case').value();
       } else if (req.url.indexOf('/documents') !== -1) {
         response = db.get('documents').value();
+      }else if (req.url.indexOf('aggregated/caseworkers/:uid/jurisdictions') !== -1) {
```

Comment:
This is part of a future task in this epic: https://tools.hmcts.net/jira/browse/RDM-3918

Reply:
Done